lib/Marlin/Marlin/src/module/temperature.cpp

```diff
@@ -3846,9 +3846,14 @@ void Temperature::isr() {
     #endif
     #if HAS_HEATED_CHAMBER
       SERIAL_ECHOPAIR(" C@:", getHeaterPower(H_CHAMBER));
-    #elif HAS_CHAMBER_API()
+    #endif
+    #if HAS_CHAMBER_API()
       auto current_chamber_temperature = buddy::chamber().current_temperature();
-      if (current_chamber_temperature.has_value()) SERIAL_ECHOPAIR(" C@:", current_chamber_temperature.value());
+      auto target_chamber_temperature = buddy::chamber().target_temperature();
+      if (current_chamber_temperature.has_value()){
```

Comment:
Nitpick/tip: You can do `if(auto temp = buddy::chamber().current_temperature())`

Reply:
In this case, we only want to print the chamber temperature if it has a value. The if on std::optional doesn't seem to check if it does contain a value, thus this isn't applicable. Otherwise this would have been great :)